tools/docs/manager.ts

```diff
@@ -1,8 +1,28 @@
+import { DateTime } from 'luxon';
 import type { RenovateConfig } from '../../lib/config/types';
+import { logger } from '../../lib/logger';
 import { getManagers } from '../../lib/modules/manager';
+import * as hostRules from '../../lib/util/host-rules';
+import { GithubHttp } from '../../lib/util/http/github';
+import { getQueryString } from '../../lib/util/url';
 import { readFile, updateFile } from '../utils';
+import type { GithubApiQueryResponse, ItemsEntity } from './github-query-items';
 import { getDisplayName, getNameWithUrl, replaceContent } from './utils';
 
+const gitHubApiUrl = 'https://api.github.com/search/issues?';
+
+if (process.env.GITHUB_TOKEN) {
+  hostRules.add({
```

Comment:
```suggestion
  logger.debug('Using GITHUB_TOKEN from env');
  hostRules.add({
```

just some vissibillity ?